.test-defs/GardenletLandscaper.yaml

```
@@ -0,0 +1,23 @@
+kind: TestDefinition
+metadata:
+  name: gardenlet-landscaper
+spec:
+  owner: gardener-oq@listserv.sap.com
+  description: Tests the Gardenlet landscaper
+  activeDeadlineSeconds: 7200
+
+  command: [bash, -c]
+  args:
+  - >-
+    go test -timeout=0 -mod=vendor ./test/integration/landscaper/gardenlet
+    --v -ginkgo.v -ginkgo.progress -ginkgo.noColor
+    -verbose=debug
+    -kubecfg=$TM_KUBECONFIG_PATH/gardener.config
+    -seed-kubecfg-path=$TM_KUBECONFIG_PATH/seed.config
+    -landscaper-kubecfg-path=$TM_KUBECONFIG_PATH/landscaper.config
+    -landscaper-target-namespace=$LANDSCAPER_NAMESPACE
+    -componentBaseUrl=$COMPONENT_DESCRIPTOR_BASE_URL
```

lets keep the naming in line with the CICD naming
```suggestion
    -repositoryContextUrl=$REPOSITORY_CTX_BASE_URL
```

---

thanks, I renamed all the flags to remove the camel casing.